modules/ossm-customize-smmr-cluster-wide.adoc

```diff
@@ -0,0 +1,26 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-create-smcp.adoc
+////
+
+:_content-type: CONCEPT
+[id="ossm-customize-smrr-cluster-wide_{context}"]
+= Customizing the member roll for cluster-wide mesh
+
+In cluster-wide mode, when you create the `ServiceMeshControlPlane` resource, the `ServiceMeshMemberRoll` resource is also created. You can modify the `ServiceMeshMemberRoll` resource after it gets created. Once you modify the resource, the {SMProductShortName} operator no longer changes it. If you modify the `ServiceMeshMemberRoll` resource using the {product-title} web console accept the prompt that indicates the modifications may be overwritten.
+
+Alternatively, you can create a `ServiceMeshMemberRoll` resource before deploying the `ServiceMeshControlPlane` resource. When you create then the `ServiceMeshControlPlane` the {SMProductShortName} Operator will not modify the `ServiceMeshMemberRoll`.
+
+[NOTE]
+====
+The `ServiceMeshMemberRoll` resource name must be named `default` and must be created in the same project namespace as the `ServiceMeshControlPlane` resource.
+====
+
+There are two ways to add a namespace to the mesh. You can either add the namespace by specifying its name in the `spec.members` list, or configure a set of namespace label selectors to include or exclude namespaces based on their labels.
+
+[NOTE]
+====
+Regardless of how members are specified in the `ServiceMeshMemberRoll` resource, you can also add members to the mesh by creating the `ServiceMeshMember` resource in each namespace.
+====
+
+See xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-member-roll-create_ossm-create-mesh[Creating the Red Hat OpenShift Service Mesh member roll] for more information.
```

Comment:
Remove this xref from the module and add it as an additional resource after the `include` statement in the assembly file

Reply:
*Sigh* such dilapidated technology. Done.

Reply:
After moving it, it seems so broken and disconnected from the context I simply deleted the link.